datafusion/core/src/physical_plan/file_format/parquet.rs

```diff
@@ -227,6 +228,7 @@ impl ExecutionPlan for ParquetExec {
             files: self.base_config.file_groups[partition_index].clone().into(),
             projector: partition_col_proj,
             adapter: SchemaAdapter::new(self.base_config.file_schema.clone()),
+            baseline_metrics: BaselineMetrics::new(&self.metrics, partition_index),
         };
 
         // Use spawn_blocking only if running from a tokio context (#2201)
```

Comment:
I think the issue is that the `ParquetExecStream` is not yet used by default (#2201) (#2202) by @tustvold 

I think you also have to add code that updates the row count in the code below that creates a `RecordBatchReceiverStream`

Reply:
@alamb Thanks fix row count in [ada393f](https://github.com/apache/arrow-datafusion/pull/2499/commits/ada393fde7fe9c535d63698d7aeba970a3d8d1d9)
which get 
```
ParquetExec: limit=None, partitions=[//Users/yangjiang/test-data/tchp-1g/orders/part-00003-e87df013-b3f8-493f-93c2-3da94f34e357-c000.snappy.parquet], projection=[o_totalprice, o_orderdate, o_orderpriority], metrics=[output_rows=899999,elapsed_compute=878.750494ms, spill_count=0,
```